ar_exercise/exercise.rb

```diff
@@ -33,14 +33,28 @@ class Apartment < ActiveRecord::Base
 # example: get every tenant in the DB
 all_tenants = Tenant.all
 
+binding.pry
 # get the first tenant in the DB
+  #  first_tenant = all_tenants.find(0)
+
 # get all tenants older than 65
+  # seniors = Tenant.where("age > 65", :age)
```

Comment:
The second argument here is no necessary since you're not subbing in a value into a string.

```ruby
# this works
seniors = Tenant.where("age > 65")

# if you need a variable for the age it would look like this:
age_to_check = 50
seniors = Tenant.where("age > ?", age_to_check)
```